platforms/ios/src/TangramMap/TGMapData.h

```diff
@@ -14,7 +14,9 @@
 /**
  Dictionary of feature properties keyed by their property name
  */
-typedef NSDictionary<NSString *, NSString *> FeatureProperties;
+typedef NSDictionary<NSString *, NSString *> TGFeatureProperties;
+
+@class TGMapViewController;
```

Comment:
We didn't need that before. Now `TGLabelPickResult.h` uses `TGMapData.h` for the typedef  `TGFeatureProperties`, but `TGLabelPickResult.h` is also included by `TGMapViewController.h` which, without the forward declaration, would include `TGMapData.h` including `TGMapViewController.h` including `TGLabelPickResult.h` including `TGMapData.h`... Classic inclusion cycle annoyance.